--- Makefile.orig	2014-12-01 16:43:00.238548201 -0500
+++ Makefile	2014-12-01 16:43:51.547712419 -0500
@@ -16,7 +16,7 @@
 PROGNAME    = afl
 VERSION     = 0.81b
 
-PREFIX     ?= /usr/local
+PREFIX      = /usr/local
 BIN_PATH    = $(PREFIX)/bin
 HELPER_PATH = $(PREFIX)/lib/afl
 DOC_PATH    = $(PREFIX)/share/doc/afl
@@ -77,13 +77,13 @@
 	rm -rf out_dir
 
 install: all
-	mkdir -p -m 755 $${DESTDIR}$(BIN_PATH) $${DESTDIR}$(HELPER_PATH) $${DESTDIR}$(DOC_PATH) $${DESTDIR}$(MISC_PATH)
+	mkdir -p -m 755 ${DESTDIR}$(BIN_PATH) ${DESTDIR}$(HELPER_PATH) ${DESTDIR}$(DOC_PATH) ${DESTDIR}$(MISC_PATH)
 	install -m 755 afl-gcc afl-fuzz afl-showmap $${DESTDIR}$(BIN_PATH)
-	for i in afl-g++ afl-clang afl-clang++; do ln -sf afl-gcc $${DESTDIR}$(BIN_PATH)/$$i; done
-	install -m 755 afl-as $${DESTDIR}$(HELPER_PATH)
-	ln -sf afl-as $${DESTDIR}$(HELPER_PATH)/as
-	install -m 644 docs/README docs/ChangeLog docs/*.txt $${DESTDIR}$(DOC_PATH)
-	cp -r testcases/ $${DESTDIR}$(MISC_PATH)
+	for i in afl-g++ afl-clang afl-clang++; do ln -sf afl-gcc ${DESTDIR}$(BIN_PATH)/$i; done
+	install -m 755 afl-as ${DESTDIR}$(HELPER_PATH)
+	ln -sf afl-as ${DESTDIR}$(HELPER_PATH)/as
+	install -m 644 docs/README docs/ChangeLog docs/*.txt ${DESTDIR}$(DOC_PATH)
+	cp -r testcases/ ${DESTDIR}$(MISC_PATH)
 
 publish: clean
 	test "`basename $$PWD`" = "afl" || exit 1
