diff -aur ext.orig/Makefile ext/Makefile
--- ext.orig/Makefile	2014-02-22 20:58:40.331217467 -0500
+++ ext/Makefile	2014-02-22 20:59:54.418370270 -0500
@@ -27,12 +27,12 @@
 PAPARAZZI_SRC=../..
 EXT_DIR=$(PAPARAZZI_SRC)/sw/ext
 
-PREFIX = arm-none-eabi
+PREFIX = arm-unknown-linux-gnueabihf
 
 #
 # if we can't find arm-none-eabi-gcc in the path, overwrite PREFIX with explicit path
 #
-HAVE_ARM_NONE_EABI_GCC := $(shell which arm-none-eabi-gcc)
+HAVE_ARM_NONE_EABI_GCC := $(shell which arm-unknown-linux-gnueabihf)
 
 ifeq ($(strip $(HAVE_ARM_NONE_EABI_GCC)),)
 # not found in PATH, check paparazzi toolchain
