diff --git a/configure b/configure
index 1d17235..bedfdb8 100755
--- a/configure
+++ b/configure
@@ -15631,7 +15631,7 @@ ac_link='$CXX -o conftest$ac_exeext $CXXFLAGS $CPPFLAGS $LDFLAGS conftest.$ac_ex
 ac_compiler_gnu=$ac_cv_cxx_compiler_gnu
 
 
-for ac_header in crypto++/sha.h crypto++/tiger.h crypto++/ripemd.h
+for ac_header in cryptopp/sha.h cryptopp/tiger.h cryptopp/ripemd.h
 do :
   as_ac_Header=`$as_echo "ac_cv_header_$ac_header" | $as_tr_sh`
 ac_fn_cxx_check_header_mongrel "$LINENO" "$ac_header" "$as_ac_Header" "$ac_includes_default"
diff --git a/src/rmd_hash.h b/src/rmd_hash.h
index 2c14a96..a6f4f95 100644
--- a/src/rmd_hash.h
+++ b/src/rmd_hash.h
@@ -23,7 +23,7 @@
 #if HAVE_CRYPTOPP==1
 
 #include "sha_hash.h"
-#include <crypto++/ripemd.h>
+#include <cryptopp/ripemd.h>
 
 //it's used just like SHA, so create it from SHA
 class rmd128hash : public shahash<CryptoPP::RIPEMD128> {};
diff --git a/src/sha_hash.h b/src/sha_hash.h
index 511841e..63604ab 100644
--- a/src/sha_hash.h
+++ b/src/sha_hash.h
@@ -23,7 +23,7 @@
 
 #include "hash.h"
 
-#include <crypto++/sha.h>
+#include <cryptopp/sha.h>
 
 template <class shatype>
 class shahash : public hash_func
diff --git a/src/tiger_hash.h b/src/tiger_hash.h
index 915b85b..e2c5d39 100644
--- a/src/tiger_hash.h
+++ b/src/tiger_hash.h
@@ -23,7 +23,7 @@
 #if HAVE_CRYPTOPP==1
 
 #include "sha_hash.h"
-#include <crypto++/tiger.h>
+#include <cryptopp/tiger.h>
 
 //it's used just like SHA, so create it from SHA
 class tiger192hash : public shahash<CryptoPP::Tiger> {};
