--- Makefile.orig	2014-11-21 16:15:34.947205471 -0500
+++ Makefile	2014-11-21 16:17:37.128002846 -0500
@@ -11,11 +11,11 @@
 
 install:
 	rm -rf $(OBJDIR)
-	python setup.py build -b $(OBJDIR) install
+	python2 setup.py install --root= --optimize=1
 
 install3:
 	rm -rf $(OBJDIR)
-	python3 setup.py build -b $(OBJDIR) install
+	python setup.py install --root= --optimize=1
 
 # NOTE: Newer cython can be installed by: sudo pip install --upgrade cython
 install_cython:
