diff --git a/cfg/targets/install.mk b/cfg/targets/install.mk
index 51beb3e..715b6f9 100644
--- a/cfg/targets/install.mk
+++ b/cfg/targets/install.mk
@@ -20,9 +20,6 @@ install: toxic
 
 	@echo "Installing man pages"
 	@for f in $(MANFILES) ; do \
-		if [ ! -e "$$f" ]; then \
-			continue ;\
-		fi ;\
 		section=$(abspath $(DESTDIR)/$(MANDIR))/man`echo $$f | rev | cut -d "." -f 1` ;\
 		file=$$section/$$f ;\
 		mkdir -p $$section ;\
