--- Makefile.orig	2014-02-22 21:47:24.219954958 -0500
+++ Makefile	2014-02-22 21:48:54.366352633 -0500
@@ -310,9 +310,9 @@
 	for f in `find	../doc/*.html -type f -maxdepth 1`; do \
 		install -m 644 $$f					$(DESTDIR)$(PREFIX)/share/doc/ivy-c; \
 	done
-	test -d $(DESTDIR)$(PREFIX)/man/man1 || mkdir -p $(DESTDIR)$(PREFIX)/man/man1
+	test -d $(DESTDIR)$(PREFIX)/share/man/man1 || mkdir -p $(DESTDIR)$(PREFIX)/share/man/man1
 	for f in `find	../doc/*.1 -type f -maxdepth 1`; do \
-		install -m 644 $$f					$(DESTDIR)$(PREFIX)/man/man1; \
+		install -m 644 $$f					$(DESTDIR)$(PREFIX)/share/man/man1; \
 	done
 	test -d $(DESTDIR)$(PREFIX)/share/doc/ivy-c/examples || mkdir -p $(DESTDIR)$(PREFIX)/share/doc/ivy-c/examples
 	install -m 644 version.h					$(DESTDIR)$(PREFIX)/share/doc/ivy-c/copyright
